src/scss/_reset.scss

```diff
@@ -39,6 +39,20 @@ h6 {
   overflow-wrap: break-word;
 }
 
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+ul {
```

Comment:
I just learned this myself. Something to keep in mind when we are authoring list https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns.

Basically we have to add `<ul role="list">` for Safari to recognize this in the accessibility tree as a list.